main.go

```diff
@@ -52,9 +57,75 @@ var (
 	BuildTime = "undefined"
 )
 
+// httpClientTimeout contains timeouts for an HTTP client and its requests.
+type httpClientTimeout struct {
+	// ClientTimeout is HTTP Client Timeout.
+	ClientTimeout time.Duration
+
+
+	// NetDialerTimeout is the net dialer timeout
+	NetDialerTimeout time.Duration
+}
+
+func loadTracing(v *viper.Viper, appName string) (candlelight.Tracing, error) {
```

Comment:
The general organization of caduceus' `main.go` is the `caduceus()` function followed by helper functions for it.  Can you please move the helper functions you've added (`loadTracing()`, `newArgusClientTimeout()`, and `newHTTPClient()`) to be below `func caduceus()`?

Reply:
Taken care of in latest commit.